### DISCUSSION TITLE
Multiple hatch seed points

### DISCUSSION THREAD

**Discussion by NikhilSP** [mozman/ezdxf#1047] [General]
Not sure if this is already known but just mentioning it:

I saw this in hatches code 

   def set_seed_points(self, points: Iterable[tuple[float, float]]) -> None:
        """Set seed points, `points` is an iterable of (x, y)-tuples.
        **I don't know why there can be more than one seed point.**
        All points in :ref:`OCS` (:attr:`Hatch.dxf.elevation` is the Z value)

Lets say there is a rectangle and a line bisecting the rectangle vertically into 2 squares. 
While creating hatches through pick point, the user will need to pick point in both the squares to fill the whole rectangle with hatch. Therefore it will have 2 seed points

**Comment by mozman** [comment]
Thank you for the information.
